### PR TITLE
To do list Milestone: Add and remove task

### PR DESCRIPTION
**Pull request by AmanWorku**
# Tasks Completed:

- Used Webpack to bundle the javascript code.
- Implemented proper ES6 syntaxes.
- Used ES6 modules to modularize the javascript files.
- Implemented a function to create, read, update and delete tasks.
- Used the local storage to save changes made.

### CODE REVIEW

File: src/addTask.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
```
Comment (omarsalem7) [inline src/addTask.js:3]: - [ ] Kindly avoid disabling linters and try to fix them.

File: src/completeTask.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
```
Comment (omarsalem7) [inline src/completeTask.js:3]: - [ ] As stated before, kindly remove these lines and fix them

File: src/deleteTask.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
```
Comment (omarsalem7) [inline src/deleteTask.js:3]: - [ ] Kindly avoid disable linters in all files and fix them

File: README.md [hunk 4]
```diff
@@ -59,14 +60,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (omarsalem7) [inline README.md:62]: - [ ] Kindly add a license section in your readme and make sure the link for it works correctly.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @AmanWorku 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct Github follow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [x] [OPTIONAL] It will be better if you make a message for commit start with a capital letter

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @AmanWorku , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_check the comment below_

- [ ] Your app adds up empty todos, please, try to prevent this behavior to improve the user experience.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**